launchdarkly-android-client-sdk/src/test/java/com/launchdarkly/sdk/android/ContextDataManagerListenersTest.java

```diff
@@ -78,7 +80,7 @@ public void listenerIsNotCalledAfterUnregistering() throws InterruptedException
         manager.registerAllFlagsListener(allFlagsListener);
         manager.unregisterAllFlagsListener(allFlagsListener);
 
-        manager.upsert(flag);
+        manager.upsert(LDContext.builder("aKey").build(), flag);
```

Comment:
I'm not confident in this test.

Reply:
Your spidey senses were correct.  Fixed.